styles.css

```diff
@@ -1 +1,751 @@
+}
+
+nav li a {
+  color: black;
+}
+
+.hamburger {
+  padding: 28px 20px;
+}
+
+.nav-menu {
+  background: #d3d3d3;
+  border: none;
+}
+
+h1.title {
+  font-size: 40px;
+  padding: 20px 0;
+  color: #ec5242;
+}
+
+p.title-para {
+  font-size: 16px;
+  padding: 20px 20px;
+  color: black;
+  border: 3px solid white;
+}
+
+p.title-para2 {
+  font-size: 16px;
+  padding: 20px 20px;
+  color: black;
+  background-color: white;
+  border: 1px solid white;
+}
+
+p.email-us {
+  font-size: 16px;
+  padding: 2%;
+  color: black;
+  text-align: center;
+}
+
+.work {
+  color: black;
+  font-size: 30px;
+  padding: 20px 20px;
+  margin: 20px 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.date {
+  color: black;
+  font-size: 30px;
+  padding: 20px 0;
+}
+
+.start {
+  font-size: 20px;
+  margin-bottom: 40px;
+}
+
+.highlights {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: black;
+  color: white;
+  background-image: url("./Images/pattern.png");
+  margin-top: -20px;
+}
+
+.title3 {
+  width: auto;
+  font-size: 1rem;
+  text-align: center;
+}
+
+section.about-logo,
+section.past {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 20px;
+  background-color: white;
+}
+
+img.my-logo {
+  width: 60%;
+  height: 0%;
+}
+
+hr.full {
+  width: 100%;
+  height: 1px;
+  color: black;
+}
+
+hr.small-hr {
+  color: #ec5242;
+  width: 7%;
+  border: 1px solid #ec5242;
+  margin: 10px;
+  margin-bottom: 20px;
+}
+
+.partners-hr {
+  color: #ec5242;
+  width: 4%;
+  border: 1px solid #ec5242;
+  margin: 10px;
+  margin-bottom: 20px;
+}
+
+h2.title2 {
+  font-size: 1.5rem;
+  width: 20%;
+  min-width: 60px;
+  color: #ec5242;
+  font-family: 'COCOGOOSE', sans-serif;
+  overflow: visible;
+}
+
+.join-btn {
+  display: block;
+  background-color: #ec5242;
+  color: #fff;
+  padding: 20px 30px;
+  font-weight: 600;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  margin: 20px auto;
+  margin-bottom: 35px;
+}
+
+.speakers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 0 5%;
+}
+
+h3,
+h3.program {
+  margin-top: 30px;
+}
+
+img.event1,
+img.event2 {
+  width: 100%;
+  height: 0%;
+  margin: 20px 0;
+}
+
+section.partners {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 20px;
+  background-color: #424146;
+}
+
+img.partner1,
+img.partner2,
+img.partner3,
+img.partner4,
+img.partner5 {
+  width: 15%;
+}
+
+div.partners-logos {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 20px 0;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  align-items: center;
+  justify-content: space-around;
+  height: 200px;
+  overflow: hidden;
+}
```

Comment:
Great design @waseemqureshi2020🤝

- [ ]  Kindly add a white footer to your about section of the mobile version so as to match the design given.
- Your design
![footer-about-mobile](https://user-images.githubusercontent.com/63917725/197341866-407e5c4b-290e-46c6-99c1-04841016d8f4.PNG)

- Design Given
![about-footer-design-mobile](https://user-images.githubusercontent.com/63917725/197341923-8b396159-a719-4773-8526-3a1e6405cd0e.PNG)

Reply:
I have changed the background color now.